tests/Unit/Evolution/Systems/Cce/Actions/Test_CharacteristicEvolutionBondiCalculations.cpp

```diff
@@ -188,9 +189,9 @@ SPECTRE_TEST_CASE(
   const double target_step_size = 0.01 * value_dist(gen);
 
   ActionTesting::MockRuntimeSystem<metavariables> runner{
-      {start_time, l_max, number_of_radial_points,
-       std::make_unique<::TimeSteppers::RungeKutta3>(),
-       std::make_unique<InitializeJ::InverseCubic>()}};
+      {start_time, std::make_unique<InitializeJ::InverseCubic>(), l_max,
```

Comment:
(optional, just a note in case you weren't aware of this) You can pass an explicit `tuples::TaggedTuple<TheTags...>{...}` to construct the runner, and it will reorder the tags for you. Then you don't have to care in which order the global cache tags happen to be.